infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -93,5 +93,14 @@ export class AtatWebApiStack extends cdk.Stack {
     // Allow the POST function to read and write (since that will be necessary to add the
     // new quotes)
     table.grantReadWriteData(postPortfolioDraftsFn);
+
+    const portfolioDrafts2 = portfolioDrafts.addResource("{portfolioDraftId}");
```

Comment:
```suggestion
    const portfolioDraft = portfolioDrafts.addResource("{portfolioDraftId}");
```

Maybe since once we've used the `{portfolioDraftID}` we're dealing with _one_ portfolio? We could even call these `portfolioDraftsRoute` and `portfolioDraftRoute` if that helps?

Reply:
Thanks for looking at this.  I was pulled into meetings this afternoon.